lib/main.dart

```diff
@@ -25,8 +25,9 @@ class App extends StatelessWidget {
       enableLog: true,
       home: SplashView(),
       supportedLocales: [
-        Locale('en', 'US'),
-        Locale('de', 'DE'),
+        Locale('en', ''),
+        Locale('de', ''),
+        Locale('fr', ''),
```

Comment:
Is it okay not to include the country code?

Reply:
I think by not including the country code you make the language available for all German/English/French speaking countries.